misc/dbt-materialize/dbt/adapters/materialize/connections.py

```diff
@@ -35,7 +42,65 @@ class MaterializeConnectionManager(PostgresConnectionManager):
 
     @classmethod
     def open(cls, connection):
-        connection = super().open(connection)
+        if connection.state == "open":
```

Comment:
I'm copying in the implementation of the [`PostgresConnectionManager`'s `open`](https://github.com/fishtown-analytics/dbt/blob/develop/plugins/postgres/dbt/adapters/postgres/connections.py#L74) here, and adding the ssl bits.

Reply:
Totally agree! I will do that next.

Reply:
🤞🏼